IdentityCore/src/network/MSIDHttpRequest.m

```diff
@@ -29,6 +29,9 @@
 #import "MSIDHttpRequestConfiguratorProtocol.h"
 #import "MSIDHttpRequestTelemetry.h"
 
+static NSInteger const s_defaultRetryCounter = 1;
+static NSTimeInterval const s_defaultRetryInterval = 0.5;
```

Comment:
what's our default retry interval in ADAL? 0.5 seconds sounds very short.

Reply:
currently it is 0.5s for ADAL (https://github.com/AzureAD/azure-activedirectory-library-for-objc/blob/dev/ADAL/src/request/ADWebAuthResponse.m#L262)